tests/modules/test_patch.py

```diff
@@ -76,7 +76,7 @@ def test_create_patch_no_change(self):
         module_path = Path(self.pipeline_dir, "modules", REPO_NAME, BISMARK_ALIGN)
 
         # Check that no patch file has been added to the directory
-        assert set(os.listdir(module_path)) == {"main.nf", "meta.yml", "environment.yml"}
+        assert "bismark-align.diff" in set(os.listdir(module_path))
```

Comment:
this is not the same assertion as was there before 🤔

Reply:
```suggestion
        assert "bismark-align.diff" not in set(os.listdir(module_path))
```